.github/workflows/check.yml

```diff
@@ -51,20 +47,17 @@ jobs:
       - sre
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          components: clippy
-          override: true
-      - name: Install Rust components and targets
+      - name: Setup toolchain
         run: |
-          rustup target add wasm32-unknown-unknown
-          rustup component add rustfmt --toolchain nightly-x86_64-unknown-linux-gnu
-          rustup target add wasm32-unknown-unknown --toolchain nightly
+         curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+         echo "$HOME/.cargo/bin:$PATH" >> $GITHUB_PATH
+      - run: |
+          rustup show
+          rustup override unset
+          rustup show
```

Comment:
I'll leave this in for now; and @seunlanlege and @haroldsphinx can have a look at why it is needed/remove it. The rustup override unset is required atm